frontend/model/contracts/group.js

```diff
@@ -701,7 +701,7 @@ sbp('chelonia/defineContract', {
           // we were getting the same latestHash upon re-logging in for test "user2 rejoins groupA".
           // We add it to the same queue as '/remove' above gets run on so that it is run after
           // contractID is removed. See also comments in 'gi.actions/identity/login'.
-          sbp('okTurtles.eventQueue/queueEvent', contractID, ['gi.actions/identity/saveOurLoginState'])
+          sbp('chelonia/queueInvocation', contractID, ['gi.actions/identity/saveOurLoginState'])
```

Comment:
I've made this selector in my PR as well. Since this is likely going to be merged first, I can merge your changes into my branch once this goes through.